scania-digital-design-system/sdds
# fix(sdds-breadcrumb): import/class/a11y fixes and renamed item

Added missing import for focus state. Changed the previous `<ol>` and `<li>` for role list/listitem for a11y. Corrected the class that is set on the last breadcrumb to remove the ::after arrow.

Also renamed the breadcumb-item to breadcumb-link. To conform with other components.

**Solving issue**  
Fixes: 

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Breadcrumb
3. Check that the styling is correct.

**Suggested test steps**

**Screenshots**  

**Additional context**
